app/assets/stylesheets/modules/buttons-links.css.scss

```diff
@@ -1,22 +1,22 @@
 .filters-button {
   background-color: $brand-primary;
-  height: 33px;
-  line-height: 33px;
-  border-radius: 30px;
-  font-size: $small-font;
   border: none;
+  border-radius: 30px;
   color: white;
-  text-align: center;
+  font-size: $small-font;
   font-weight: $font-stack-bold;
+  height: 33px;
+  text-align: center;
 }
 
-.circular-button {
+.link-button {
   height: 33px;
+  margin-top: 3px;
   width: 33px;
-}
 
-.link-button {
-  marign-top: 3px;
+  img {
+    vertical-align: middle;
```

Comment:
fixes alignment. note: changing line-height messes with this, but there's no reason to set line-height if no text